schemas/firefox-accounts/account-ecosystem/account-ecosystem.1.schema.json

```diff
@@ -12,6 +12,16 @@
       ]
     }
   ],
+  "mozPipelineMetadata": {
+    "bq_dataset": "firefox_accounts",
+    "bq_table": "account_ecosystem_v1",
```

Comment:
This template schema doesn't define these bq fields; they are default values injected by `inject_metadata.py` based on the directory structure.